### PR TITLE
[INFRA-2348] - Create a top-level documentation for the Jenkins Core Release Flow

### PR DESCRIPTION
**Pull request by oleg-nenashev**
Just a summary of my repositories and review.
https://issues.jenkins-ci.org/browse/INFRA-2348

CC @olblak @MarkEWaite

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+which would be able to release Jenkins Weekly and LTS versions in an automatic way.
+It includes packaging, staging, and releasing stages.
+
+## Jenkins Core Release Environment
+
+The release environment is hosted within the https://github.com/jenkins-infra/openvpn[Jenkins VPN].
+It includes a Jenkins instance and a number of other services.
+If you are connected to VPN, the Jenkins instance can be accessed using this URL: https://release.ci.jenkins.io/ .
+
+Helm Chart for the Jenkins Core Release Environment is available https://github.com/jenkins-infra/charts/tree/master/charts/jenkins[here].
```
Comment (olblak) [inline README.adoc]: Thi link https://github.com/jenkins-infra/charts/blob/master/helmfile.d/jenkins-release.yaml makes more sense as it includes the jenkins chart and the release environlment charts

File: README.adoc [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+The following components can be found here:
+
+* Jenkinsfiles for Jenkins packaging and Release
+* Kubernetes POD templates for Linux and Windows packaging environments
+* Kubernetes POD template for the release environment
+* Release profiles and utility scripts
+
+## Jenkins Packaging
+
+* WAR packaging is defined winthin the https://github.com/jenkinsci/jenkins[Jenkins core] repository.
```
Comment (olblak) [inline README.adoc]: https://github.com/jenkinsci/packaging but my PR is not yet merged

File: README.adoc (hunk 1 above)
Comment (oleg-nenashev) [reply README.adoc]: Will add it before the environment chart sentence

Comment (oleg-nenashev) [reply README.adoc]:
```suggestion
Chart for Jenkins Core Release Environment environment is available link: https://github.com/jenkins-infra/charts/blob/master/helmfile.d/jenkins-release.yaml[here].
Jenkins master chart for it is available https://github.com/jenkins-infra/charts/tree/master/charts/jenkins[here].
```

File: README.adoc [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+which would be able to release Jenkins Weekly and LTS versions in an automatic way.
+It includes packaging, staging, and releasing stages.
+
+## Jenkins Core Release Environment
+
+The release environment is hosted within the https://github.com/jenkins-infra/openvpn[Jenkins VPN].
+It includes a Jenkins instance and a number of other services.
+If you are connected to VPN, the Jenkins instance can be accessed using this URL: https://release.ci.jenkins.io/ .
+
+Chart for Jenkins Core Release Environment environment is available link: https://github.com/jenkins-infra/charts/blob/master/helmfile.d/jenkins-release.yaml[here].
```
Comment (timja) [inline README.adoc]:
```suggestion
Chart for Jenkins Core Release Environment environment is available link:https://github.com/jenkins-infra/charts/blob/master/helmfile.d/jenkins-release.yaml[here].
```

File: README.adoc (hunk 2 above)
Comment (oleg-nenashev) [reply README.adoc]: Not sure I understand it. WAR package itself is generated within the core at the moment. I can update the doc to reference jenkinsci/packaging as a source of codesigning scripts which run on the top of it

Comment (olblak) [reply README.adoc]: Yes are totally right, jenkinsci/packaging assume that a war file is already "released"

**Comment by oleg-nenashev** [comment]
some docs are better than no docs. Thanks for reviews!

**Pull request merged**